### PR TITLE
Rename threads

### PR DESCRIPTION
**Pull request by breeswish**
## What have you changed? (mandatory)

Some thread names are too long, some don't have a unified format. This PR renames these threads to follow the same convention. Notice that txn scheduler is not changed.

Update: Here is a list of thread names after applying this PR:

```
  TID COMMAND
16868 tikv-server
16873 tikv-server
16875 tikv-server
16876 tikv-server
16877 pd-0
16879 tikv-server
16880 time-monitor
16887 addr-resolver
16890 rocksdb:bg0
16891 rocksdb:bg1
16892 rocksdb:bg2
16894 rocksdb:bg3
16896 rocksdb:bg4
16897 rocksdb:bg0
17051 store-read-high
17052 store-read-high
17053 store-read-high
17054 store-read-high
17055 store-read-norm
17056 store-read-norm
17057 store-read-norm
17058 store-read-norm
17059 store-read-low0
17060 store-read-low1
17061 store-read-low2
17062 store-read-low3
17063 sst-importer0
17064 sst-importer1
17065 sst-importer2
17066 sst-importer3
17067 sst-importer4
17068 sst-importer5
17069 sst-importer6
17070 sst-importer7
17071 cop-high0
17072 cop-high1
17073 cop-high2
17074 cop-high3
17075 cop-high4
17076 cop-high5
17077 cop-high6
17078 cop-high7
17079 cop-normal0
17080 cop-normal1
17081 cop-normal2
17082 cop-normal3
17083 cop-normal4
17084 cop-normal5
17085 cop-normal6
17086 cop-normal7
17087 cop-low0
17088 cop-low1
17089 cop-low2
17090 cop-low3
17091 cop-low4
17092 cop-low5
17093 cop-low6
17094 cop-low7
17095 grpc-server-0
17096 grpc-server-1
17097 grpc-server-2
17098 grpc-server-3
17099 debugger0
17100 raftstore-1
17101 promepusher
17102 split-check
17103 snap-generator
17104 snap-generator
17105 snapshot-worker
17106 raft-gc-worker
17107 compact-worker
17108 pd-worker
17109 consistency-chk
17110 cleanup-sst
17111 apply-worker
17112 sched-worker-po
17113 sched-worker-po
17114 sched-worker-po
17115 sched-worker-po
17116 sched-high-pri-
17117 storage-schedul
17118 gc-worker
17119 rocksdb-metrics
17120 end-point-worke
17121 snap-sender0
17122 snap-sender1
17123 snap-sender2
17124 snap-sender3
17125 snap-handler
17129 tikv-server
17132 tikv-server
```

## What are the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

no.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

no.

## Does this PR affect tidb-ansible update? (mandatory)

https://github.com/pingcap/tidb-ansible/pull/474

## Refer to a related PR or issue link (optional)

Close https://github.com/pingcap/tikv/issues/3347

### CODE REVIEW

**Comment by siddontang** [comment]
I don't like the name of `wkr`, what does it mean?

**Comment by siddontang** [comment]
so I suggest removing worker, the thread name is readable even without `worker` as the suffix.

**Comment by siddontang** [comment]
or you must think out a more readable name

**Comment by breeswish** [comment]
@siddontang @huachaohuang Ok, I will change it.

File: src/util/rocksdb/metrics_flusher.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl MetricsFlusher {
         let interval = self.interval;
         self.sender = Some(tx);
         let h = Builder::new()
-            .name(thd_name!("rocksb-metrics-flusher"))
+            .name(thd_name!("rdb-metric-flush"))
```
Comment (zhangjinpeng1987) [inline src/util/rocksdb/metrics_flusher.rs]: First look at rdb, let me connect to redis' rdb.

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -221,14 +221,14 @@ impl<T: Transport, C: PdClient> Store<T, C> {
...
-            compact_worker: Worker::new("compact worker"),
+            split_check_worker: Worker::new("split-check"),
+            region_worker: Worker::new("snapshot-worker"),
+            raftlog_gc_worker: Worker::new("raft-gc-worker"),
+            compact_worker: Worker::new("compact-worker"),
             pd_worker,
-            consistency_check_worker: Worker::new("consistency check worker"),
-            cleanup_sst_worker: Worker::new("cleanup sst worker"),
-            apply_worker: Worker::new("apply worker"),
+            consistency_check_worker: Worker::new("consistency-chk"),
```
Comment (siddontang) [inline src/raftstore/store/store.rs]: consistency-checker

Comment (breeswish) [reply src/raftstore/store/store.rs]: That's ok, although the name overflows 🤣

Comment (siddontang) [reply src/raftstore/store/store.rs]: use full name at first

**Pull request merged**